spec/factories/case/offender_sar_complaints.rb

```diff
@@ -54,27 +54,51 @@
       recipient { 'requester_recipient' }
     end
 
-    trait :data_to_be_requested do
+    trait :to_be_assessed do
       # Default state for a new offender_sar_complaint
     end
 
+    trait :data_review_required do
+      transient do
+        identifier { 'Data review required - Complaint' }
+      end
+
+      after(:create) do |kase|
+        create :case_transition_data_review_required, case: kase
+        kase.reload
+      end
+    end
+
+    trait :data_to_be_requested do
+      transient do
+        identifier { 'Data to be requested - Complaint' }
+      end
+
+      after(:create) do |kase|
+        create :case_transition_data_to_be_requested, case: kase
+        kase.reload
+      end
+    end
+
     trait :waiting_for_data do
       transient do
-        identifier { 'Waiting for data Offender SAR' }
+        identifier { 'Waiting for data - Complaint' }
```

Comment:
Just thinking it may be good to have these identifiers as 'Offender SAR Complaint' rather than just complaint (who knows we may have other complaint types in future and just complaint may be a little ambiguous. Also we consistently refer to Offender Sar Complaints within the application 👍